### DISCUSSION TITLE
Home Keys

### DISCUSSION THREAD

**Discussion by MauricioNO** [HomeSpan/HomeSpan#192] [General]
Hello guys!
Apparently Apple recently launched, with iOS 15, a new feature called Home keys, more can be found here on Apple's official website: https://support.apple.com/guide/iphone/unlock-your-door-with-a-home-key-iph0dc255875/ios

This is **very new** stuff, so not much seems to be known around yet, however I wanted to check here if you guys know anything about it, and maybe a possible (future) use with HomeSpan. From what I gathered, some lock mechanisms will have the option to generate a Home key (a card in Apple wallet) as a way to unlock it.

It would be very cool to be able to use the iPhone to unlock our projects :)

Thanks!

**Comment by HomeSpan** [comment]
It looks like a great feature but unfortunately Apple has not released any details in the HAP documentation.  It does not appear Apple is continuing to support the non-commercial versions of HAP (upon which HomeSpan, HomeBridge, and the Apple as well as Espressif HomeKit SDKs are based).  If you come across any information please let me know.

**Reply by MauricioNO** [reply]
I’ll absolutely share it here if I learn more. Hopefully they do pick back up on supporting the non-commercial version. 

Thank you!
